flume-ng-sinks/flume-ng-elasticsearch-sink/src/main/java/org/apache/flume/sink/elasticsearch/ElasticSearchSink.java

```diff
@@ -331,13 +330,8 @@ public void configure(Context context) {
       indexNameBuilder = clazz.newInstance();
       indexnameBuilderContext.put(INDEX_NAME, indexName);
       indexNameBuilder.configure(indexnameBuilderContext);
-    } catch (Exception e) {
-      logger.error("Could not instantiate index name builder.", e);
-      Throwables.propagate(e);
-    }
-
-    if (sinkCounter == null) {
-      sinkCounter = new SinkCounter(getName());
```

Comment:
Note to reviewers: SportBugs was reporting `CN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE` because `sinkCounter` is instantiated twice.